docs/app/Component/ComponentDescription.js

```diff
@@ -58,10 +59,10 @@ class ComponentDescription extends Component {
             </a>
           </code>
         </div>
-        {this.isSemanticComponent && semanticDocsLink}
-      </div>
+        <p className='sub header'>
+          {this.isSemanticComponent && semanticDocsLink}
+        </p>
+      </Segment>
```

Comment:
The `ComponentDescription` is the top section of every component doc section.  It is now a basic segment so that it has a bottom margin.  This is needed to keep whitespace between the header and prop types table.